dist/assets/css/custom.css

```diff
@@ -289,19 +289,29 @@
 .portfolio-1 .portfolio-item .portfolio-link .details .description {
     background-color: #3e444d;
     color: white;
-    display: table;
-    height: 30px;
+    /*display: table;*/
+    height: 72px;
     list-style: none;
     margin-bottom: 0;
     width: 100%;
     font-size: 14px;
     padding: 8px 12px;
 }
+.portfolio-1 .portfolio-item .portfolio-link .details .description:after {
+  content: "";
+  text-align: right;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  width: 25%;
+  height: 20px;
+  background: linear-gradient(to right, rgba(62, 68, 77, 0), rgba(62, 68, 77, 1) 50%);
```

Comment:
The other sections appear to be using 4 space indent but this one is using 2 space indent, is that intentional?